packages/api/portfolios/environments/createEnvironment.test.ts

```diff
@@ -18,6 +18,21 @@ describe("createEnvironment", () => {
     console.log(result?.body);
     expect(result?.statusCode).toBe(SuccessStatusCode.CREATED);
   });
+  it.skip("requests with properties from BaseObject should cause status code 400", async () => {
+    const validRequest: ApiGatewayEventParsed<IEnvironmentCreate> = {
+      body: {
+        name: "flaw check 1",
+        id: "7bc938ca-4c1c-4740-8ccf-18c940a70865",
+        createdAt: "2020-12-17T18:29:42.769Z",
+        updatedAt: "2020-12-17T18:29:42.769Z",
+        archivedAt: "2099-12-17T18:29:42.769Z",
+      },
```

Comment:
Just noticing in my own branch that this should be otherwise a valid body so the 400 isn't caused by not including required properties.

Reply:
Thanks, for pointing that out. Updated to ensure it is a valid request and only errors for the added properties.